src/cwts/networkanalysis/run/RunNetworkLayout.java

```diff
@@ -0,0 +1,454 @@
+          + "    weights.\n"
+          + "--sorted-edge-list\n"
+          + "    Indicates that the edge list file is sorted. The file should be sorted based\n"
+          + "    on the nodes in the first column, followed by the nodes in the second\n"
+          + "    column. Each edge should be included in both directions in the file.\n"
+          + "--input-layout <filename> (default: random layout)\n"
+          + "    Read the initial layout from the specified file. The file is expected to\n"
+          + "    contain three tab-separated columns (without a header line), first a column\n"
+          + "    of nodes, then a column of x coordinates, and finally a column of\n"
+          + "    y coordinates. Nodes are represented by zero-index integer numbers. If no\n"
+          + "    file is specified, a random layout (in which each node is positioned at\n"
+          + "    random coordinates) is used as the initial layout.\n"
+          + "-o --output-layout <filename> (default: standard output)\n"
+          + "    Write the final layout to the specified file. If no file is specified,\n"
+          + "    the standard output is used.\n";
+
+    /**
+     * This method is called when the tool is started.
+     *
+     * @param args Command line arguments
+     */
+    public static void main(String[] args)
+    {
+        System.err.println(DESCRIPTION);
+
+        // Process command line arguments.
+        if (args.length == 0)
+        {
+            System.err.print(USAGE);
+            System.exit(-1);
+        }
+
+        int normalization = DEFAULT_NORMALIZATION;
+        int attraction = DEFAULT_ATTRACTION;
+        int repulsion = DEFAULT_REPULSION;
+        int nRandomStarts = DEFAULT_N_RANDOM_STARTS;
+        int maxNIterations = DEFAULT_MAX_N_ITERATIONS;
+        double initialStepSize = DEFAULT_INITIAL_STEP_SIZE;
+        double minStepSize = DEFAULT_MIN_STEP_SIZE;
+        double stepSizeReduction = DEFAULT_STEP_SIZE_REDUCTION;
+        int requiredNQualityValueImprovements = DEFAULT_REQUIRED_N_QUALITY_VALUE_IMPROVEMENTS;
+
+        long seed = 0;
+        boolean useSeed = false;
+        boolean weightedEdges = false;
+        boolean sortedEdgeList = false;
+        String initialLayoutFilename = null;
+        String finalLayoutFilename = null;
+        String edgeListFilename = null;
+
+        int argIndex = 0;
+        while (argIndex < args.length - 1)
+        {
+            String arg = args[argIndex];
+            try
+            {
+                if (arg.equals("-n") || arg.equals("--normalization"))
+                {
+                    if (((argIndex + 1) >= args.length) || (!args[argIndex + 1].equals(NORMALIZATION_NAMES[NO_NORMALIZATION]) && !args[argIndex + 1].equals(NORMALIZATION_NAMES[ASSOCIATION_STRENGTH]) && !args[argIndex + 1].equals(NORMALIZATION_NAMES[FRACTIONALIZATION]) && !args[argIndex + 1].equals(NORMALIZATION_NAMES[LINLOG])))
```

Comment:
Again, I would use a `switch ... case` here, and raise an exception in the `default` option.